src/backend/app.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const path = require("path");
+const mealsRouter = require("./api/meals");
+const reservationsRouter = require("./api/reservations");
+//The body function has changed after version 15 while I am using 16. If put request doesnt work then install bodyparser
+//const bodyOarser = require("body-parser");
+//npm install --save body-parser
+
+
+const buildPath = path.join(__dirname, "../../dist");
+const port = process.env.PORT || 3000;
+const cors = require("cors");
+
+// For week4 no need to look into this!
+// Serve the built client html
+app.use(express.static(buildPath));
+
+// Parse URL-encoded bodies (as sent by HTML forms)
```

Comment:
Try to aim for clear spacing between the different blocks of the code. That way, it will look nicer in code interviews, quizzes etc.